website/docs/how-to.md

```diff
@@ -325,8 +331,8 @@ targetFormats:
   - DELTA
 datasets:
   -
-    tableBasePath: s3://path/to/iceberg  # replace this with gs://path/to/iceberg_data if your data is in GCS.
-    tableDataPath: s3://path/to/iceberg/data
+    tableBasePath: s3://path/to/iceberg-dataset  # replace this with gs://path/to/iceberg_data if your data is in GCS.
```

Comment:
To prevent ambiguity, you could append 'people', which is the name of the table, to the path.